package.json

```diff
@@ -29,7 +29,7 @@
   "devDependencies": {
     "@andersdjohnson/tsconfig": "^1.0.1",
     "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
+    "@babel/core": "^7.18.6",
     "@babel/node": "^7.12.6",
```

Comment:
@ronderksen Do we have to include the package.json and package-lock.json files in this PR?

Reply:
Maybe not. My IDE warned about a security issue with v7.12.3 of @babel/core, so I updated it.

Reply:
@ronderksen Ah thanks, good call! I guess it could be a separate effort to update dependencies, but no worries. I'm not having as much time to maintain my older repos lately.